src/lib/auth.ts

```diff
@@ -5,17 +5,37 @@ import type { User } from "firebase/auth"
 import { app } from './firebase'
 
 export interface AuthState {
+  // auth.user will be either a firebase User object or null
```

Comment:
I'm not a fan of comments that simple re-state what the code already defines - I assume people are familiar with JS and TypeScript, and can inspect types etc...